src/components/create-guild/CreateGuildButton.tsx

```diff
@@ -20,7 +23,18 @@ const CreateGuildButton = ({ isDisabled }: Props): JSX.Element => {
       isDisabled={response || isLoading || isSigning || isDisabled}
       isLoading={isLoading || isSigning}
       loadingText={signLoadingText || "Saving data"}
-      onClick={handleSubmit(onSubmit)}
+      onClick={() => {
+        captureEvent("guild creation flow > templates selected", {
+          roles: roles.map((role) => role.name),
+        })
+        captureEvent("guild creation flow > number of platforms added to roles", {
```

Comment:
I think we could rename this to `guild creation flow > added ${platformConnected} platforms`